### PR TITLE
Adding force snapshot and force backup APIs

### PR DESCRIPTION
**Pull request by rdelval**
* Have you run goformat on the project before submitting? Yes

* Have you run go test on the project before submitting? Do all tests pass? Yes, Yes

* Does the Pull Request require a test to be added to the end to end tests? If so, has it been added? Yes, Yes

### CODE REVIEW

File: realis.go [hunk 1]
```diff
@@ -75,6 +75,8 @@ type Realis interface {
 	MaintenanceStatus(hosts ...string) (*aurora.Response, *aurora.MaintenanceStatusResult_, error)
 	SetQuota(role string, cpu *float64, ram *int64, disk *int64) (*aurora.Response, error)
 	GetQuota(role string) (*aurora.Response, error)
+	Snapshot() error
```
Comment (at15) [inline realis.go]: Actually I prefer having comment on the interface, godoc would look better and normally when people jump to definition, the first jump to the interface.

File: examples/client.go [hunk 2]
```diff
@@ -644,12 +644,28 @@ func main() {
...
-			fmt.Print("error: %+v\n", err.Error())
+			fmt.Printf("error: %+v\n", err.Error())
 			os.Exit(1)
 		}
 		fmt.Println("map size: ", len(result.Configs))
 		fmt.Println(result.String())
 
+	case "snapshot":
+		fmt.Println("Forcing scheduler to write snapshot to mesos replicated log")
+		err := r.Snapshot()
```
Comment (at15) [inline examples/client.go]: for example code, we can just `log.Fatal(err)`  which is `fmt.Printf` + `os.Exit` with timestamp

File: realis.go [hunk 3]
```diff
@@ -940,6 +942,10 @@ func (r *realisClient) RollbackJobUpdate(key aurora.JobUpdateKey, message string
 	return resp, nil
 }
 
+/* Admin functions */
+// TODO(rdelvalle): Consider moving these functions to another interface. It would be a backwards incompatible change,
```
Comment (at15) [inline realis.go]: we can have version number to distinguish this kind of breaking change, it seems we haven't bump version number for a while ... https://github.com/paypal/gorealis/releases

File: realis.go (hunk 1 above)
Comment (rdelval) [reply realis.go]: Good point, will tackle this in a different PR where I'll move all comments to the interface.

File: realis.go (hunk 3 above)
Comment (rdelval) [reply realis.go]: Agreed, I want to pile up a bunch of breaking changes into a V2. I should keep a list of this somewhere in the repo.

Comment (at15) [reply realis.go]: We can add a CHANGELOG.md in root of repo and keep those stuff under unreleased section, so people can know we have new features and use them if they want to force version using commit hash . i.e. https://github.com/gaocegege/maintainer/blob/master/CHANGELOG.md

**Pull request merged**